### PR TITLE
add wd-my-cloud cask

### PR DESCRIPTION
**Pull request by mbicz**

### CODE REVIEW

File: Casks/wd-my-cloud.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  sha256 'f0aef2aa1791bc6af9c4fb21d0b55dbeda39474449bc3770068e010fa73daebd'
+
+  url 'http://download.wdc.com/nas/WDMyCloud_mac.dmg'
+  name 'WD My Cloud'
+  homepage 'http://www.wdc.com/'
+  license :closed
+
+  pkg 'Install WD My Cloud.pkg'
+  uninstall  :pkgutil => 'com.wdc.wdMyCloud.*', :rmdir => '/Applications/WD My Cloud'
+
```
Comment (vitorgalvao) [inline Casks/wd-my-cloud.rb]: Please remove this line.

File: Casks/wd-my-cloud.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  version '1.0.7.4'
+  sha256 'f0aef2aa1791bc6af9c4fb21d0b55dbeda39474449bc3770068e010fa73daebd'
+
+  url 'http://download.wdc.com/nas/WDMyCloud_mac.dmg'
+  name 'WD My Cloud'
+  homepage 'http://www.wdc.com/'
+  license :closed
+
+  pkg 'Install WD My Cloud.pkg'
+  uninstall  :pkgutil => 'com.wdc.wdMyCloud.*', :rmdir => '/Applications/WD My Cloud'
```
Comment (vitorgalvao) [inline Casks/wd-my-cloud.rb]: There should be an empty line above this one. `:pkgutil` and `:rmdir` should also be in different lines.

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

**Comment by ndr-qef** [comment]
Thanks. Merged as 93e9e31.

**Pull request closed without merging**